packages/client/src/__tests__/integration/happy/not-so-exhaustive-schema-mongo/common.ts

```diff
@@ -9,15 +9,14 @@ if (isMacOrWindowsCI) {
   jest.setTimeout(80_000)
 }
 
-export const testGeneratedClient = (runtimeName: 'binary' | 'library' | 'dataProxy') => async () => {
+export const testGeneratedClient = (runtimeName: 'binary' | 'library') => async () => {
```

Comment:
As said above, there's only two runtimes now. In a follow up PR, we will likely remove displaying them completely.